Add `deploy-account` command (#1083)
software-mansion/protostar: website/docs/cli-reference.md

Used by whitelisted users for deploying contracts in Alpha MainNet.
#### `--wait-for-acceptance`
Waits for transaction to be accepted on chain.
### `deploy-account`

> Sends deploy-account transaction. The account contract must be already declared and prefunded.

Reviewer: ```suggestion
Deploys a new account on selected network. The account contract must be already declared and prefunded.
```

Author: I think the original version is more correct. Deployer deploys the account contract, not Protostar.

#### `--account-address STRING`

> Account address

Reviewer: ```suggestion
Account address.
```

Author: [update descriptions](https://github.com/software-mansion/protostar/pull/1083/commits/f5d59f69597d68d828ac69ea259e5c851c7b1502)
